src/Stepper/Step.jsx

```diff
@@ -0,0 +1,336 @@
+import React, {PropTypes} from 'react';
+
+import {
+  TouchRipple,
+} from '../ripples';
+
+import Avatar from '../avatar';
+import Mixins from '../mixins';
+
+import {getMuiTheme} from '../styles';
+
+const Step = React.createClass({
+  propTypes: {
+    children: PropTypes.node,
+
+    /**
+     *Override the inline-styles of div which contains all the children include control button groups
+     */
+    childrenWrapperStyle: PropTypes.object,
+    /**
+     *Override the inline-styles of connector line
+     */
+    connectorLineStyle: PropTypes.object,
+    /**
+     *An array of node for handling moving or canceling steps
+     */
+    controlButtonsGroup: PropTypes.arrayOf(PropTypes.node),
```

Comment:
Okay I understand, I think there's an approach where we could get the best of both worlds. We might be able to inspect the props of an incoming element, and merge its `style` into your predefined defaults. Maybe there's a better way too. Let's see what the others say.